src/library/Uno.Material/Styles/Application/Colors.xaml

```diff
@@ -29,22 +29,15 @@
 			<Color x:Key="OverlayColor">#51000000</Color>
 
 			<!-- TODO: Temporary until we figure out how to propertly set Theme Colors in BottomNavigationBar -->
-			<StaticResource x:Key="BottomNavBackgroundColorBrush"
-							ResourceKey="PrimaryBrush" />
+			<Color x:Key="BottomNavBackgroundColor">#5B4CF5</Color>
 
 			<!-- TODO: Temporary until we figure out how to propertly set Theme Colors in BottomNavigationBarItem -->
-			<StaticResource x:Key="BottomNavLabelFocusColorBrush"
-							ResourceKey="OnPrimaryBrush" />
-			<StaticResource x:Key="BottomNavIconFocusColorBrush"
-							ResourceKey="OnPrimaryBrush" />
-			<StaticResource x:Key="BottomNavLabelColorBrush"
-							ResourceKey="OnPrimaryMediumBrush" />
-			<StaticResource x:Key="BottomNavIconColorBrush"
-							ResourceKey="OnPrimaryMediumBrush" />
-			<StaticResource x:Key="BottomNavNotifBackgroundColorBrush"
-							ResourceKey="ErrorBrush" />
-			<StaticResource x:Key="BottomNavNotifLabelColorBrush"
-							ResourceKey="OnErrorBrush" />
+			<Color x:Key="BottomNavLabelFocusColor">#FFFFFF</Color>
+			<Color x:Key="BottomNavIconFocusColor">#FFFFFF</Color>
+			<Color x:Key="BottomNavLabelColor">#FFFFFF</Color>
+			<Color x:Key="BottomNavIconColor">#FFFFFF</Color>
+			<Color x:Key="BottomNavNotifBackgroundColor">#F85977</Color>
+			<Color x:Key="BottomNavNotifLabelColor">#000000</Color>
```

Comment:
We can close this PR like this temporily but here's what I think would work better (and avoid duplicating hex values)

```
			<SolidColorBrush x:Key="BottomNavBackgroundColorBrush"
							 Color="{ThemeResource PrimaryColor}"/>
			
			<SolidColorBrush x:Key="BottomNavLabelFocusColorBrush"
							 Color="{ThemeResource OnPrimaryColor}"/>

			<SolidColorBrush x:Key="BottomNavIconFocusColorBrush"
							 Color="{ThemeResource OnPrimaryColor}"/>

			<SolidColorBrush x:Key="BottomNavIconFocusColorBrush"
							 Color="{ThemeResource PrimaryColor}"/>
			
			<SolidColorBrush x:Key="BottomNavBackgroundColorBrush"
							 Color="{ThemeResource PrimaryColor}"/>
			<SolidColorBrush x:Key="BottomNavBackgroundColorBrush"
							 Color="{ThemeResource PrimaryColor}"/>
```

Reply:
Yes I completely agree, we need to remove this soon.